week2/README.md

```diff
@@ -0,0 +1 @@
+This is a demo README.md created on May 31, 2024. 
```

Comment:
There isn't anything wrong here, but I wanted to post a review comment to show you the interface for marking them as resolved. 

Typically, if I suggest a change, you can discuss it here and mark it as resolved once you've addressed it. 

There isn't anything to change here, so you can go ahead and mark it as resolved below.